styles.css

```diff
@@ -356,7 +370,8 @@ hr {
   font-size: 72px !important;
```

Comment:
- As requested by the previous reviewer in this [review link](https://github.com/Xghayor/Portfolio/pull/6#discussion_r1231412715) you are yet to remove all of the instances where you have used the `!important` css property. 
- [ ] * According to the Microverse list for [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), You have overused the `!important` rule in your CSS style definitions which is not allowed and not a good practice, except you are using some CSS framework to add your styles. As this appears to be in multiple lines in your CSS style definitions, please, kindly _**REMOVE ALL THE `!important` RULE FROM YOUR CODEBASE**_ and use another approach to add the desired rule you want to meet this part of the CSS requirement.
- > please make this required changes and resubmit to be approved :+1: :sparkles:

Reply:
@Efiamotu-1 
i was using only five, does it really exceed the limit?